java/src/jmri/jmrit/display/logixng/LogixNGBundle.properties

```diff
@@ -101,3 +104,15 @@ LogixNG action:\n\
 Module: {1}\n\
 Action: {2}\n
 
+WindowToFront_HideOrShow_Show       = Show window
+WindowToFront_HideOrShow_Hide       = Hide window
+WindowToFront_HideOrShow_DoNothing  =
+
+WindowToFront_MaximizeMinimizeNormalize_Minimize    = Minimize window
+WindowToFront_MaximizeMinimizeNormalize_Normalize   = Normalize window
+WindowToFront_MaximizeMinimizeNormalize_Maximize    = Maximize window
+WindowToFront_MaximizeMinimizeNormalize_DoNothing   =
+
+WindowToFront_BringToFrontOrBack_Front      = Bring to front
+WindowToFront_BringToFrontOrBack_Back       = Bring to back
+WindowToFront_BringToFrontOrBack_DoNothing  =
```

Comment:
@dsand47 
All these three selections have the option `Do nothing`. I wasn't sure if it's best to have the option empty or to have the text `Do nothing`. I'm open to both.

Reply:
The empty selection for do nothing works for me.